src/common/sha2/sha2_c.c

```diff
@@ -691,8 +709,10 @@ void oqs_sha2_sha256_inc_finalize_c(uint8_t *out, sha256ctx *state, const uint8_
 	if (new_inlen == inlen) {
 		new_in = in;
 	} else { //Combine incremental data with final input
-		tmp_in = OQS_MEM_checked_malloc(tmp_len);
-
+		tmp_in = OQS_MEM_malloc(tmp_len);
```

Comment:
lots of instances with this pattern: Why "externalize" the ==NULL check dropping the warning output in OQS_MEM_checked_malloc? This does not provide more code safety but only less user information, no?

Reply:
fixed by OQS_EXIT_IF_NULLPTR